openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -31,6 +30,12 @@
 
 log = logging.getLogger(__name__)
 
+# Loading pynliner is actually quite expensive because it uses cssutils, which
+# has an expensive initialization. All told, it accounts for 15-20% of "fast"
+# LMS startup (without asset compilation). So we're going to set this as `None`
+# to start, and then lazily replace with the actual import when it's needed.
+pynliner = None  # pylint: disable=invalid-name
```

Comment:
I still feel like a Python newbie.  This seemed like interesting code for lazy loading the import.  I searched on Google and [turned up this link](http://stackoverflow.com/questions/4177735/best-practice-for-lazy-loading-python-modules).  In this case, is that link not accurate, and is there additional processing each time import is called?  Or, am I missing something else?  Thanks!